index.d.ts

```diff
@@ -37,10 +37,6 @@ export interface Action {
  * side-effects. This is what enables exciting features like hot reloading and
  * time travel.
  *
- * Reducers are the most important concept in Redux.
- *
- * *Do not put API calls into reducers.*
```

Comment:
This is far too broad a statement.  As long as those APIs are also pure and side-effect free there is no reason not to use them in a reducer.  (For example, for handling complex data arithmetic one might want to use the `moment` API.)